docs/specs/assets.md

```diff
@@ -20,3 +20,13 @@
 Image formats supported vary depending on the player. Some commonly supported formats are JPEG, GIF, PNG and SVG. 
 
 {schema_object:assets/image}
+
+If the dimensions of the image asset does not match the size given by `w` and `h`,
+renderers MUST ensure image layers referencing that asset do not have any visuals
+exceeding the `w`-`h` size. It's RECOMMENDED they scale the image maintaining
+its aspect ratio and they center it within the $(0, 0)$, $(w, h)$ box.
+
+Even if an image asset does not have any intrinsic size, its contents MUST
+still stay within the `w`-`h` bounds when rendered.
+
+Authoring tools SHOULD export files where `w` and `h` match the physical size of the assets.
```

Comment:
Ye, it's stated here:

> It's RECOMMENDED they scale the image maintaining its aspect ratio and they center it within the $(0, 0)$, $(w, h)$ box.

Reply:
That's just the player recommendation, right? I'm talking about the case where a user wants to author a Lottie where the image display dimensions are different than the file dimensions. Should we explicitly recommend authoring tools accomplish this using a layer scale transform?

Reply:
IMO there's no need to specify that, the recommendation is to export `w`,`h` as per the asset, and image layers can still be scaled so that's the only way of following the recommendation.

I don't think we need to specify every use case / feature interaction.